include/jdksavdecc_jdks.h

```diff
@@ -577,9 +572,101 @@ static inline ssize_t
 
 extern struct jdksavdecc_eui64 jdksavdecc_jdks_aem_control_ipv4_parameters;
 
-/*@}*/
```

Comment:
Why is this being removed? It should be changed to /*@{*/. See Doxygen docs.

Reply:
[jdksavdecc_jdks.zip](https://github.com/audioscience/jdksavdecc-c/files/1287010/jdksavdecc_jdks.zip)
Made corrections based on above points. Please review.

Reply:
Should i add on patch over commit? Or should wait for your feedback.

Reply:
Just create another commit and push it to your local repo and the pull request will grab it.

Reply:
Done. Sorry got delayed a bit.. Same file which was shared for post review is added here.